environment.yml

```diff
@@ -1,3 +1,4 @@
+name: ngff-latency-benchmarking
```

Comment:
I'll add for now, but the only downside of including `name:` is that then it's hard to make two versions locally.

Reply:
I will roll it back and adjust command accordingly